packages/config/config/devices/0x0330/zv9001k4-dim.json

```diff
@@ -0,0 +1,46 @@
+// Sunricher ZV9001K4-DIM
+// Z-Wave Dim Remote Control
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV9001K4-DIM",
+	"description": "Z-Wave Dim Remote Control",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0xa310"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Factory Reset",
+			"description": "Factory reset",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 21930,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
```

Comment:
In other devices that have a factory reset parameter, we set `allowManualEntry` to `false` and provide two options - one for the default state and one for the value to factory reset. Please do the same here (and in the other file above).

Reply:
I've looked trough the manufacturer documentation, and it actually doesn't state any factory reset parameter. I've based this on the [Z-Wave Alliance docs](https://products.z-wavealliance.org/products/3783?selectedFrequencyId=1) of the SR-ZV9001K4-DIM-G2, but I think the information there is incorrect. So I've just opted to remove it instead.